lib/github_api/params_hash.rb

```diff
@@ -9,7 +9,7 @@ class ParamsHash < DelegateClass(Hash)
     include Normalizer
     include MimeType
 
-    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers]
+    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers, :options]
```

Comment:
See I got confused about this since it was the only way I could see to pass in a header. But if I didn't specify this `:options` here then it appended it to the URL params like `?header="Accept: some gibberish"` which isn't desirable.